website_twitter_no_ext_links/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================================================================
+Twitter Feeds and user stats with widget support and no external links
+======================================================================
+
+A twitter extension that includes:
+
+    * Multiple Twitter Accounts and Feeds.
+    * Customized theming of every twitter Account.
+    * Widget that allows multiple twitter accounts on the same page.
+    * Statistics on most mentioned users in every account.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+
+# Go to Knowledge>Collaborative Content>Configure Twitter Account 
+create a new Account (API Key and secret needed).
+    the account needs to have:
+        1) a name
+        2) a default theme
+        3) a user to fetch from
+
+# Go to Knowledge>Collaborative Content>Twitter Website Widget Themes 
+and create a new theme, or just use one of the default ones.
+        
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. go to the website and drag a the new "Twitter (no links)"
+   widget in the desired area.
+
+#. Choose in widget options twitter feed and theme.
+
+#. save
+
+Known issues / Roadmap
+======================
+
+TODO:
+    Create a widget
+    Active links in twitter feed?
+    widget to visualize most referenced partners in a cloud.
+
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/website/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/website/issues/new?body=module:%20website_twitter_simple%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+
```

Comment:
find out if we're allowed to use the twitter logo in the first place, and if so, name your source here